python/tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = py27,py36,py37
+skipdist = true
+
+[testenv]
+deps =
+        -rrequirements.txt
```

Comment:
I just added -rrequirements.txt into the requirements-dev.txt file. So we reference only the dev file in tox.ini

Reply:
Thanks. I will update this eventually once https://github.com/combust/mleap/pull/569 has been merged.